src/test/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClientTest.java

```diff
@@ -33,12 +33,7 @@
 import io.searchbox.client.config.ElasticsearchVersion;
 import io.searchbox.client.config.HttpClientConfig;
 import io.searchbox.cluster.NodesInfo;
-import io.searchbox.core.BulkResult;
-import io.searchbox.core.Delete;
-import io.searchbox.core.Index;
-import io.searchbox.core.Search;
-import io.searchbox.core.SearchResult;
-import io.searchbox.core.Update;
+import io.searchbox.core.*;
```

Comment:
our convention is not to use *

Reply:
Sorry, my IDE did that automatically. I'll fix that.